website/tutorial/creating-an-extension.md

```diff
@@ -0,0 +1,342 @@
+---
+sidebar_position: 8
+title: Creating an extension
+description: Creating a Podman Desktop extension
+keywords: [podman desktop, podman, extension]
+tags: [podman-desktop, creating-an-extension]
+---
+
+# Creating a Podman Desktop extension
+
+This tutorial covers the following end-to-end tasks required to create and run a Podman Desktop extension:
+
+- Initializing an extension
+- Writing the extension entry point
+- Build dependencies
+- Running the extension
+- Verifying the extension's features
```

Comment:
In the documentation those links where clickable, any reason why this is not the case here?